client/src/game/Game.js

```diff
@@ -23,10 +53,12 @@ const Game = (props) => {
 
     gameLoop(canvas);
 
-    window.addEventListener("keydown", handleKeyDown);
+    window.addEventListener("keydown", handleKeyDown, true);
```

Comment:
What is `true` used for?

Reply:
https://stackoverflow.com/questions/5657292/why-is-false-used-after-this-simple-addeventlistener-function

The 3rd param is called `useCapture` and related to the children of the element. [useCapture = true, The handler is set on the capturing phase. Events will get to it before getting to its children.] My canvas element doesn't have children, so I don't need it ;)